.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+#vscode
+.vscode
```

Comment:
- Push a .env file template (with no confidential information) to github then add the .env file here since we don't want the confidential information we shall set in our machines for our project to be available to any one who has access to our github repo.
- Let the template have the name `.env-sample` then add some comments on the readme.MD file on how to update the  .env file when someone is trying to set up our project for the first time.